File: librarymanager/librarymanager-be/src/main/java/ru/spring5/elena/librarymanager/api/dto/Book.java
package ru.spring5.elena.librarymanager.api.dto;

import lombok.Builder;
import lombok.Getter;

@Builder
@Getter
public class Book {
    private Long bookId;
    private boolean isReserved;
    private boolean isAvailable;
    private int shelfNumber;
    private String title;
    private Long editionId;
}

File: librarymanager-api/src/main/java/ru/elenacherev/librarymanager/api/dto/Book.kt
package ru.elenacherev.librarymanager.api.dto

import java.util.*

data class Book(
    var bookId: UUID? = null,
    val isReserved: Boolean = false,
    val isAvailable: Boolean = false,
    val shelfNumber: Int = 0,
    val title: String,
    val editionId: UUID
)